lms/djangoapps/certificates/views/webview.py

```diff
@@ -473,10 +474,12 @@ def render_cert_by_uuid(request, certificate_uuid):
     template_path="certificates/server-error.html",
     test_func=lambda request: request.GET.get('preview', None)
 )
+@pluggable_override('RENDER_CERTIFICATE_VIEW_IMPL')
 def render_html_view(request, course_id, certificate=None):
     """
     This public view generates an HTML representation of the specified user and course
     If a certificate is not available, we display a "Sorry!" screen instead
+    It can be overridden by setting `RENDER_CERTIFICATE_VIEW_IMPL` to an alternative implementation.
```

Comment:
I see that the docstring, from which this has been copied, was not updated after the reworks. I'll fix this in #27466. Please use the prefix `OVERRIDE_` for this (e.g. [OVERRIDE_GET_UNIT_ICON](https://github.com/edx/edx-platform/blob/068ac5290f0e5dcd770b789d766b2cd5d22a1376/openedx/core/lib/xblock_utils/__init__.py#L558)) to keep it consistent.

Reply:
Thanks for pointing that out. I've updated the variable name here to match.